packages/flutter/test/services/binding_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:convert';
```

Comment:
Do I have to add some copyright header?

Reply:
Yes, just copy it from one of the other files.